### ISSUE TITLE
BUG: org.rogach.scallop.exceptions.UnknownOption: Unknown option '	1'

### ISSUE THREAD

**Issue by emchristiansen** [scallop/scallop#40]
I don't think this code should throw an exception, but I'm new to scallop so I could be wrong.

To reproduce, compile and run this code:
"""
case class Foo(x: String)

class Conf(args: Seq[String]) extends ScallopConf(args) {
  val xs = opt[List[String]](default = Some%28Nil%29) map (_ map Foo.apply) 
}

object Main {
  def main(unparsedArgs: Array[String]) {
    val args = new Conf(unparsedArgs)
    def foos: List[Foo] = args.xs.get.get
    // This line causes a crash.
    println(foos)
  }
}
"""

This produces this error:
Exception in thread "main" org.rogach.scallop.exceptions.UnknownOption: Unknown option '    1'
    at org.rogach.scallop.Scallop$$anonfun$get$6.apply(Scallop.scala:489)
    at org.rogach.scallop.Scallop$$anonfun$get$6.apply(Scallop.scala:489)
    at scala.Option.getOrElse(Option.scala:120)
    at org.rogach.scallop.Scallop.get(Scallop.scala:489)
    at org.rogach.scallop.ScallopConf$$anon$1.fn$lzycompute(ScallopConf.scala:124)
    at org.rogach.scallop.ScallopConf$$anon$1.fn(ScallopConf.scala:124)
    at org.rogach.scallop.ScallopOption.get(ScallopOption.scala:23)
    at org.rogach.scallop.ScallopOption$$anon$4.fn$lzycompute(ScallopOption.scala:82)
    at org.rogach.scallop.ScallopOption$$anon$4.fn(ScallopOption.scala:82)
    at org.rogach.scallop.ScallopOption.get(ScallopOption.scala:23)
    at tangram.Main$.foos$1(main.scala:27)
    at tangram.Main$.main(main.scala:29)
    at tangram.Main.main(main.scala)

**Comment by Rogach** [comment]
What arguments do you pass into that snippet? (what is the value of `unparsedArgs`)?

**Comment by emchristiansen** [comment]
I believe I get the error regardless of which args I pass

**Comment by Rogach** [comment]
Yes, that indeed is a bug - the option name guesser failed. If you tweak your option definition line to this:

``` scala
val xs = opt[List[String]]("xs", default = Some(Nil)) map (_ map Foo.apply)
```

the example would work as expected.

**Comment by Rogach** [comment]
I published the snapshot with the fix (for Scala 2.9.x):

```
resolvers += "sonatype snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
libraryDependencies += "org.rogach" %% "scallop" % "0.8.0~1~SNAPSHOT"
```

By the way, you don't need to call `args.xs.get.get` - `args.xs()` would be shorter and has the same effect.

**Comment by Rogach** [comment]
Does it work for you now?
